make_dockerfiles.py

```diff
@@ -57,6 +57,9 @@
 repo_version['rowan']       = versions['ROWAN_VERSION']       = 'v1.2.2'
 shas['ROWAN_SHA'] = '88ed2c0003543d3154374f741a04fcb21cce32180e7b29152dc93c4c887e0c51'
 
+repo_version['coxeter']       = versions['ROWAN_VERSION']       = 'v0.2.0'
```

Comment:
```suggestion
repo_version['coxeter']       = versions['COXETER_VERSION']       = 'v0.2.0'
```

Reply:
Thanks I missed one in the find-replace.